purchases/src/test/java/com/revenuecat/purchases/PostingTransactionsTests.kt

```diff
@@ -359,7 +361,7 @@ class PostingTransactionsTests {
     }
 
     @Test
-    fun `consumeAllTransactions is sent as observerMode when posting to backend`() {
+    fun `!consumeAllTransactions is sent as observerMode when posting to backend`() {
```

Comment:
I thought the `!` was a typo. Can we rename this test to something similar to:

```suggestion
    fun `observerMode parameter is sent as false when transactions are consumed`() {
```